### PR TITLE
Update everything

### PR DESCRIPTION
**Pull request by rvsia**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
     "sass-loader": "^8.0.2",
     "source-map-loader": "^1.0.0",
     "style-loader": "^1.2.1",
-    "stylelint": "^13.5.0",
+    "stylelint": "^13.6.0",
```
Comment (Hyperkid123) [inline package.json:85]: Now that I see it, is stylelint even used?

Comment (rvsia) [reply package.json:85]: There is a script for it, but it's not used in CI and I also do not use it.

Comment (Hyperkid123) [reply package.json:85]: I wonder if we could just get rid of it

Comment (rvsia) [reply package.json]: I tried it and it actually does nothing = it's removed.

**Pull request merged**